android/src/main/java/com/swmansion/gesturehandler/core/NativeViewGestureHandler.kt

```diff
@@ -208,6 +220,14 @@ class NativeViewGestureHandler : GestureHandler<NativeViewGestureHandler>() {
     fun sendTouchEvent(view: View?, event: MotionEvent) = view?.onTouchEvent(event)
   }
 
+  private class TextViewHook() : NativeViewGestureHandlerHook {
+    override fun shouldRecognizeSimultaneously(handler: GestureHandler<*>) = false
+
+    // We have to explicitly check for ReactTextView, since its `isPressed` flag is not set to `true`,
+    // in contrast to e.g. Touchable
+    override fun canActivate(view: View) = view.isPressed || view is ReactTextView
```

Comment:
And this one for view being text?

Reply:
Also done in [df1c2aa](https://github.com/software-mansion/react-native-gesture-handler/pull/3202/commits/df1c2aae26e3eebd10861f3b7f57e19994458b87)